AEPMedia.xcodeproj/project.pbxproj

```diff
@@ -1137,8 +1296,11 @@
 				MTL_ENABLE_DEBUG_INFO = NO;
 				MTL_FAST_MATH = YES;
 				SDKROOT = iphoneos;
+				SUPPORTED_PLATFORMS = "iphonesimulator iphoneos appletvos appletvsimulator";
```

Comment:
do we need to have SDKROOT defined for lib targets? some of our other repos don't seem to have it (same for the debug target above)

Reply:
yes we have Base SDK configured for all